src/loader/LoaderUtils.ts

```diff
@@ -82,23 +149,6 @@ export async function loadTextureAtlas(
   return textureAtlas;
 }
 
-export function createTextureAtlas(atlasText: string, textures: Texture2D[]) {
-  const textureAtlas = new TextureAtlas(atlasText);
-  textureAtlas.pages.forEach((page, index) => {
-    const engineTexture = textures.find(item => item.name === page.name) || textures[index];
-    const texture = createAdaptiveTexture(engineTexture);
-    page.setTexture(texture);
-  });
-  return textureAtlas;
-}
-
-export function loadTexture(
-  url: string, 
-  engine: Engine,
-  imageLoaderType: string = AssetType.Texture2D
-): AssetPromise<Texture2D> {
-  return engine.resourceManager.load<Texture2D>({ url, type: imageLoaderType });
-}
 
 export function createAdaptiveTexture(texture: Texture2D) {
   return new AdaptiveTexture(new Image(), texture);
```

Comment:
Is it necessary to encapsulate this function?

Reply:
Done. Deleted.